CONTRIBUTING.md

```diff
@@ -14,30 +14,30 @@ team](https://github.com/orgs/woodpecker-ci/teams/advisors/members).
 Of course, if an advisor has time to code review, we will gladly welcome them back to the maintainers team.
 If a maintainer is inactive for more than 3 months and forgets to leave the maintainers team, the owners may move him or her from the maintainers team to the advisors team.
 
-For security reasons, Maintainers must use 2FA for their accounts and if possible provide GPG signed commits.  
-https://help.github.com/articles/securing-your-account-with-two-factor-authentication-2fa/
-https://help.github.com/articles/signing-commits-with-gpg/
+For security reasons, Maintainers must use 2FA for their accounts and if possible provide GPG signed commits.
+<https://help.github.com/articles/securing-your-account-with-two-factor-authentication-2fa/>
+<https://help.github.com/articles/signing-commits-with-gpg/>
 
 ## Owners
 
 Since Woodpecker is a pure community organization without any company support,
-to keep the development healthy we will elect two owners every year.  
```

Comment:
that duble empty spaces at the end are intentional newlines!!!

Reply:
I know. But IMO one of the worst inventions ever done given the cleanup of whitespaces in almost any other language.
Linebreaks also do the job, i.e. one sentence per line.
Such trailing whitespaces are often not even shown in editors unless the config is explicitly defined to do so.

How about we just clean up all MD docs and go with "one sentence per line"?

Reply:
There's also the option to append a `\` at the end which is not affected by this.

Reply:
I would vote for `\` as its visible regardless of editor settings.